staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/google/cel-go/cel"
 	"github.com/google/cel-go/checker"
 	"github.com/google/cel-go/common/types"
+	"github.com/pkg/errors"
```

Comment:
oh... don't use pkg/errors, use stdlib `errors`, and drop the go.mod change

Reply:
Updated and here is the changes into current release: https://github.com/kubernetes/kubernetes/pull/126308